game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -28,7 +28,7 @@
  */
 @Log
 public class GameSelectorModel extends Observable {
-  @Getter private GameData gameData = null;
+  @Nullable @Getter private GameData gameData = null;
```

Comment:
GameData may be nulllable, marking it allows us to get IDE warnings for potential NPE